### PR TITLE
Added big example to top of CommentThread class — with nesting

### PR DESCRIPTION
**Pull request by jarednova**
**Ticket**: #1971

## Issue
As noted, the only example of how to do nested comments is buried in the tests

## Solution
Put an example in!

## Impact
Expands the example in `Timber\Post::comments` showing a nested example. The same example appears in the a new docblock for `Timber\CommentThread`.

## Considerations
Is this the right place? Originally I had it just in CommentThread — but since that's not really intended to be a public class, it felt wrong. This solution provides where I think people are most likely to discover it: with `{{ post.comments }}` which they interface with, and `CommentThread.php` in case someone is looking through the files

### CODE REVIEW

File: lib/CommentThread.php [hunk 1]
```diff
@@ -54,6 +91,19 @@ public function order( $order = 'ASC' ) {
 		return $this;
 	}
 
+	/**
+	 * @internal
+	 * @experimental
```
Comment (gchtr) [inline lib/CommentThread.php]: The `@experimental` tag is not really an official tag, or is it?

Comment (jarednova) [reply lib/CommentThread.php]: No, those are something I saw at some point and brought over — but not official. I think we should scrub them all (and probably the associated functions in 2.0)

**Comment by jarednova** [comment]
@palmiak can I get a follow-up review on this? I added info on using `comment-reply.js` in this update so we should be totally covered now

File: lib/CommentThread.php (hunk 1 above)
Comment (gchtr) [reply lib/CommentThread.php]: I just added a [project card](https://github.com/timber/timber/projects/4#card-22187846) for this.

**Comment by jarednova** [comment]
@palmiak yes. My hope is that this can finally be put to 🛏 — thanks!

**Pull request merged**